core/ErrorQueue.h

```diff
@@ -30,24 +29,21 @@ class ErrorQueue {
     std::atomic<int> filesFlushedCount{0};
     bool ignoreFlushes{false};
 
-    ErrorQueue(spdlog::logger &logger, spdlog::logger &tracer);
-    ~ErrorQueue();
+    ErrorQueue(spdlog::logger &logger, spdlog::logger &tracer,
```

Comment:
(Optional) I think we can remove `tracer` and use `gs.tracer()` in all of the places it's currently used.